decode_morse.rb

```diff
@@ -0,0 +1,57 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def decode_char(str)
+  if MORSE_CODE.key?(str)
+    MORSE_CODE[str]
+  else
+    ''
+  end
+end
+
+def decode_word(str)
+  arr = []
+  str.split.each do |n|
+    arr.push(MORSE_CODE[n])
+  end
+  arr.join
+end
```

Comment:
Thank you for using the DRY rule for the `#decode` method.
`#decode_word` should also follow the DRY rule and reuse the `#decode_char` method to translate a single char symbol into a letter.

Reply:
Its not possible please

Reply:
Thank you.
We have implemented the change and now #decode_word is following the DRY rule and reusing the #decode_char method to translate a single char symbol into a letter.